tests/kernel/queue/src/test_queue_contexts.c

```diff
@@ -14,20 +14,20 @@ K_QUEUE_DEFINE(kqueue);
 K_MEM_POOL_DEFINE(mem_pool_fail, 4, 8, 1, 4);
 K_MEM_POOL_DEFINE(mem_pool_pass, 4, 64, 4, 4);
 
-struct k_queue queue;
-static qdata_t data[LIST_LEN];
-static qdata_t data_p[LIST_LEN];
-static qdata_t data_l[LIST_LEN];
-static qdata_t data_sl[LIST_LEN];
+APP_BSS_MEM struct k_queue queue;
```

Comment:
Why is a kernel object going into app memory?

Reply:
Actually, why was this file changed at all?
All the tests in here run in supervisor mode, it's only test_queue_user.c that has user mode tests